blogposts/2022/sourcegraph-extension-for-vs-code.md

```diff
@@ -48,8 +48,8 @@ gradle org\.apache\.logging\.log4j ['"] 2\.(0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|1
 As you can see in the video below, the global search returns several repositories that require further investigation. I finally changed the search context to Sourcegraph to narrow down my search scope to find all the repositories affecting our organization.
 
 <video loop autoplay muted playsinline>
-  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.webm" type="video/webm" />
-  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.mp4" type="video/mp4" />
+  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.webm" type="video/webm" data-cookieconsent="ignore" />
+  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.mp4" type="video/mp4" data-cookieconsent="ignore" />
```

Comment:
The only one that doesn't work is this one which we are already aware of

Reply:
Yeah, let's circle back and try to get to the bottom of that as a follow up!